### PR TITLE
Fix invalid receipts error with rest function call

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Charge gas amount for skipped REST function calls
- Delete timer flags file for testnet-dev
- Turn off console logging for local trackers

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1185

### CODE REVIEW

File: db/functions.js [hunk 1]
```diff
@@ -255,43 +253,51 @@ class Functions {
...
+                  }
+                  failCount++;
+                  return true;
+                }));
+                funcResults[functionEntry.function_id] = {
+                  code: FunctionResultCode.SUCCESS,
+                  bandwidth_gas_amount: blockchainParams.restFunctionCallGasAmount,
+                };
+              } else {
+                // Rest function trigger is skipped.
```
Comment (minsulee2) [inline db/functions.js:295]: Checked!! Thanks :)

File: common/result-code.js [hunk 2]
```diff
@@ -126,6 +126,7 @@ const FailedTxPrecheckCodeSet = new Set([
 // If they are altered and deployed, the full sync of the blockchain nodes can fail.
 const FunctionResultCode = {
   SUCCESS: 0,
+  SKIP: 20000,  // Normal skip
```
Comment (minsulee2) [inline common/result-code.js:129]: 👍

**Review by minsulee2 — APPROVED** [review]
LGTM! Thanks for the fast fix :)

**Review by ehgmsdk20 — APPROVED** [review]
LGTM, thanks!

**Comment by platfowner** [comment]
Thanks for the review. Now merging.. @minsulee2 @ehgmsdk20

**Pull request merged**